tests/beman/inplace_vector/compare.test.cpp

```diff
@@ -11,6 +11,10 @@ concept has_threeway = requires(const T &t) {
   { t <=> t };
 };
 
+template <class T>
+concept lessthan_comparable =
+    beman::details::inplace_vector::lessthan_comparable<T>;
```

Comment:
It strikes me that the namespace is backwards here -- should be beman::inplace_vector::detail -- but that's an existing issue of course. And maybe something we don't address clearly in the standard.

Reply:
I might be wrong here, but beman::inplace_vector clashes with the inplace_vector struct under the beman namespace.